cmd/contact-auditor/main.go

```diff
@@ -140,22 +154,34 @@ func (c contactAuditor) run(resChan chan *result) error {
 	return nil
 }
 
+func makeDBConnection() (*sql.DB, error) {
+	conf, err := mysql.ParseDSN(vars.DBConnSAMailer)
+	conf.Params = make(map[string]string)
+
+	// Transaction isolation level READ UNCOMMITTED trades consistency
+	// for performance.
+	conf.Params["tx_isolation"] = "'READ-UNCOMMITTED'"
```

Comment:
For this case I would use a map litera:

```
conf.Params = map[string]string{
  "tx_isolation": "'READ-UNCOMMITTED'",
}
```

Also I'm surprised this needs the inner `'`. Any idea why?

Reply:
Thanks for explaining!